example/web/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>example</title>
+</head>
+<body>
+  <script src="main.dart.js" type="application/javascript"></script>
+</body>
+</html>
```

Comment:
extra file?

Reply:
This is actually the base for the web project, without which you can't run in web mode. So I will leave it.